content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -118,18 +122,27 @@ The cluster status is updated to **Deleting** while cluster resources are being
 
 <br />
 
-## Force Delete a Cluster
 
-If a cluster is stuck in the Deletion state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scopes. To force delete a cluster, follow the same steps outlined in [Delete a MAAS Cluster](/clusters/data-center/maas/create-manage-maas-clusters#deleteamaascluster). After 15 minutes, a **Force Delete Cluster** option is available in the **Settings drop-down menu**. The drop-down menu will provide you with an estimated remaining duration before the force deletion becomes available.
+# Upgrading a MAAS Cluster
 
-<br />
+Upgrading a cluster involves careful planning and executing the upgrade while minimizing disruptions to your applications and services. Spectro Cloud provides an intuitive and streamlined upgrade process through the MAAS, simplifying the entire process.
+
+To learn more about managing a cluster, refer to [Manage Clusters](https://docs.spectrocloud.com/clusters/cluster-management/cluster-updates).
```

Comment:
```suggestion
To learn more about managing a cluster, refer to [Manage Clusters](https://docs.spectrocloud.com/clusters/cluster-management/cluster-updates).

<br />
```

Add a breakline between content and a box element.

Reply:
Done.